content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,231 @@
+- Show how much XRP is set aside for the account's [reserve requirement](reserves.html). 
+- Send [direct XRP payments](direct-xrp-payments.html), and provide feedback about the intended destination address, including:
+    - Displaying your account's available balance
+    - Verifying that the destination address is valid
+    - Validating the account has enough XRP to send
+    - Allowing you to specify a destination tag
+
+## Steps
+
+Before you begin, make sure you have the pre-requisites installed. Check your node version by running `node -v`. If you don't have node installed, you can download it [here](https://nodejs.org/en/download/).
+
+### Step 1: Setting up the project
+
+1. Navigate to the directory that you want to create the project in.
+2. Run 
+   ```sh
+    yarn create vite-app <project-name>
+    ```
+    or
+    ```sh
+    npm init vite-app <project-name>
+    ```
+    to create a new project.
+
+3. Navigate to your project directory and copy this [package.json](_code-samples/build-a-wallet/js/package.json) file to your project.
+        - Alternatively you can also do `yarn add <package-name>` or `npm install <package-name>` for each individual package to add them to your `package.json` file.
+  
+4. Run `yarn` or `npm i` to install the packages.
+
+5. Create a new file `.env` in the root directory of the project and add the following variables:
+
+```bash
+CLIENT="wss://s.altnet.rippletest.net/" // or any other server
+EXPLORER_NETWORK="testnet" // or "mainnet"
+SEED="sEd7B2Jm7RBhhh5i88SH38aZB8ot8Kq" // Replace with your seed
+```
+6. Change the seed to your own seed. You can get a testnet account from [here](https://xrpl.org/xrp-test-net-faucet.html).
+
+### Step 2: Creating the Home Page (Displaying Account & Ledger Details)
+
+In this step, we will create the home page that will display the account details and the ledger details.
+
+![Home Page Screenshot](img/js-wallet-home.png)
+
+1. Make a new folder named src in the root directory of the project.
+
+2. Create new files named index.html, index.js and index.css. 
+
+3. Copy the contents of the [index.html](_code-samples/build-a-wallet/js/index.html), [index.js](_code-samples/build-a-wallet/js/index.js) and [index.css](_code-samples/build-a-wallet/js/index.css) files to your files.
+
+4. Create a new file in helpers folder named get-wallet-details.js and copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/helpers/get-wallet-details.js", language="js") }}
+
+In the function getWalletDetails, we are using the [account_info](https://xrpl.org/account_info.html) method to fetch the account details. We are also using the [server_info](https://xrpl.org/server_info.html) method to calculate the reserve base and increment. You can read more about the reserve requirement [here](https://xrpl.org/reserves.html). Follow the comments in the code to understand the logic.
```

Comment:
Also, instead of saying "follow the comments" after the code, you'll probably want to say that beforehand. Otherwise the reader would have to scroll down, then back up, as opposed to just paying attention as they scroll down :)

Reply:
Removed, English is the hardest part of coding :p